src/environment.rs

```diff
@@ -94,22 +97,25 @@ impl SystemEnvironment {
 }
 
 impl RCCEnvironment {
-    pub fn bundled_command_spec(binary_path: &Utf8Path) -> CommandSpec {
+    pub fn bundled_command_spec(binary_path: &Utf8Path, robocorp_home: String) -> CommandSpec {
```

Comment:
Let's make this an instance method instead and use `self.robocorp_home`

Reply:
This is very awkward in those places, which don't have any `RCC` plan (such as turning of telemetry). We would have to make  all fields of `RCCEnvironment` optional. I think some our abstractions do not work.